crates/telio-relay/src/derp/http.rs

```diff
@@ -204,24 +260,38 @@ async fn connect_http<R: AsyncRead + Unpin, W: AsyncWrite + Unpin>(
     writer: &mut W,
     server_keepalives: &DerpKeepaliveConfig,
     host: &str,
+    derp_version: DerpVersion,
 ) -> Result<Vec<u8>, Error> {
-    writer
-        .write_all(
-            format!(
-                "GET /derp HTTP/1.1\r\n\
+    let request = match derp_version {
+        // We try to connect to derp V2
+        DerpVersion::V2 => format!(
+            "GET /v2/derp HTTP/1.1\r\n\
                 Host: {host}\r\n\
                 Connection: Upgrade\r\n\
                 Upgrade: WebSocket\r\n\
                 User-Agent: telio/{} {}\r\n\
-                Keep-Alive: tcp={}, derp={}\r\n\r\n",
-                telio_utils::version_tag(),
-                std::env::consts::OS,
-                server_keepalives.tcp_keepalive,
-                server_keepalives.derp_keepalive,
-            )
-            .as_bytes(),
-        )
-        .await?;
+                Poll-Keepalive: {}\r\n\r\n",
+            telio_utils::version_tag(),
+            std::env::consts::OS,
+            server_keepalives.derp_keepalive,
+        ),
```

Comment:
This is duplicated with below - only path changes. Please remove the duplication.